docs/README.md

```diff
@@ -517,14 +520,14 @@ export class PostReadModel {
 
 #### 6. Deployment
 
-At this point, we've learned:
+At this point, we've:
 
-- how to create a publicly accessible command
-- we emitted an event to store the data
-- we reduced the event into an entity
-- and finally, we projected the entity into a read model that is also publicly accessible.
+- Created a publicly accessible command
+- Emitted an event as a mechanism to store data.
+- Reduced the event into an entity to have a representation of our internal state
+- Projected the entity into a read model that is also publicly accessible.
 
-That's all; you already know the basics to build event-driven, CQRS-based applications
+With this, you already know the basics to build event-driven, CQRS-based applications
```

Comment:
To be honest, I prefer `That's all`, as it somehow concludes and rewards the reader.

Reply:
![image](https://user-images.githubusercontent.com/2271237/88172332-b66b7b00-cc18-11ea-996a-0c2b8403d0ce.png)

Reply:
Now seriously. If you read the previous paragraph, `That's all` seems like it doesn't fit well.